e2etest/e2etest_suite_test.go

```diff
@@ -169,43 +138,29 @@ var _ = ginkgo.BeforeSuite(func() {
 	framework.ExpectNoError(err)
 	v4Addresses := strings.Split(ipv4ForContainers, ",")
 	v6Addresses := strings.Split(ipv6ForContainers, ",")
-	frrContainers, err = setupContainers(v4Addresses, v6Addresses)
+	bgptests.FRRContainers, err = bgptests.SetupContainers(v4Addresses, v6Addresses, cs)
```

Comment:
Having global parameters is already sub optimal, but we can't do much about that because the tests entry points are not functions we can invoke.
Functions called by other packages that change them as a side effect is imo less clear than having them explicitly modified, that's why I tried to keep SetupContainers pure and without state.